packages/react-core-auth/package.json

```diff
@@ -33,10 +33,12 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@aws-amplify/core": "6.4.0",
```

Comment:
Wasn't sure what version to use, went with 6.4.0 because it's the latest that would be used by the  `"aws-amplify": "^6.3.2"` peer dep